### DISCUSSION TITLE
Baghdad exploit

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#949] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So on interstellar this new exploit called Baghdad anyone tried this? Does it work for 121?

**Comment by cloneyfirey** [comment]
where is it?

**Reply by v4pl** [reply]
IN the TN sever

**Reply by cloneyfirey** [reply]
i dont see it. Which channel in the server

**Reply by v4pl** [reply, marked as answer]
Oh the interstellar sever i mean

**Comment by Quartinal** [comment]
@vaperbest, do you mean my exploit https://github.com/Quartinal/Baghdad? I wouldn't be surprised if it works on 121, but I'm on 120 and it works.

**Reply by v4pl** [reply]
Crazy lucky you

**Reply by CGRmags** [reply]
you see I would try it but I'm on 120 lol but I will have my friend try it and I will tell you Ok? @vaperbest

**Reply by Quartinal** [reply]
@vaperbest, are you using 52.207.185.90? This DNS isn't hosted anymore, 150.136.163.0 might work for you; it works for me.

**Reply by v4pl** [reply]
Oh thanks

**Reply by v4pl** [reply]
Are you the creator?

**Reply by v4pl** [reply]
It only last 1 minute btw right?

**Reply by Quartinal** [reply]
Yes, Brandon421-ops posted a similar exploit (Incognito On The Sign-In screen), but the steps are not exactly the same, so I am the creator of Baghdad.

**Reply by Quartinal** [reply]
It may last one minute, though every computer varies, so I cannot say for certain.

**Reply by v4pl** [reply]
It only last for seconds for me

**Reply by Quartinal** [reply]
It takes practice, this was the same for me at first.

**Reply by v4pl** [reply]
Do you know about the Omaha dns?

**Reply by Quartinal** [reply]
Yes, if I understand correctly, I read about it at https://caudns.vercel.app/#:~:text=Credits%20to%20Caub%20and%20OmadaDNS.

**Reply by xoidex** [reply]
hey @Quartinal, wasn't this exploit discovered by @thecoolmiker? [original post](https://github.com/3kh0/ext-remover/discussions/737)
also there's a way to prevent it from disappearing

**Reply by Quartinal** [reply]
He did, it's just the steps aren't accurate because the incognito DNS changed (to 150.136.163.0). I don't update Baghdad a lot. And yes, there are in fact multiple ways to do the exploit and to prevent it from disappearing. Much of the reason I made Baghdad was to make it available to people for future updates and to make a helper. The new steps for ChromeOS 120 and 121, and furthermore [SWAB](https://github.com/Quartinal/SWAB), were discovered by me.

**Reply by xoidex** [reply]
oh cool
